bindata/v4.1.0/kube-apiserver/pod.yaml

```diff
@@ -238,7 +238,7 @@ spec:
       path: /etc/kubernetes/static-pod-resources/kube-apiserver-pod-REVISION
     name: resource-dir
   - hostPath:
-      path: /etc/kubernetes/static-pod-resources/kube-apiserver-certs
+      path: /etc/kubernetes/static-pod-resources/kube-apiserver-pod-REVISION/kube-apiserver-certs
```

Comment:
cert-sync-certs maybe?

Reply:
or unrevisioned ?

Reply:
or cert-sync

Reply:
or dynamic

Reply:
@mfojtik join the bike shedding

Reply:
+1

as of today we have
```
ls /etc/kubernetes/static-pod-resources/kube-apiserver-pod-18/kube-apiserver-certs
configmaps  secrets
```

Reply:
I like `dynamic`

Reply:
The name `dynamic` is controlled by the operator. So we would have to change it in the other places as well.